lib/internal/modules/package_json_reader.js

```diff
@@ -64,6 +71,7 @@ function deserializePackageJSON(path, contents) {
       ObjectDefineProperty(this, 'exports', { __proto__: null, value });
       return this.exports;
     },
+    ...(everything && contents[6]),
```

Comment:
Can we be explicit? Depending on the package.json this can overwrite existing values in this object